objects/obj_controller/Create_0.gml

```diff
@@ -181,8 +181,7 @@ large_text=ini_read_real("Settings","large_text",0);
 settings_heresy=ini_read_real("Settings","settings_heresy",0);
 settings_fullscreen=ini_read_real("Settings","fullscreen",1);
 settings_window_data=ini_read_string("Settings","window_data","fullscreen");
-        if (test_map == 1)
-    global.cheat_debug = 1
+if (is_test_map=true) then global.cheat_debug=true;
```

Comment:
you can actually have `if (is_test_map) ...`, I personally would use this shorter version, but it's a very minor thing, up to you

Reply:
Well, tbh my knowledge of coding is very limited ,but thanks to you and Zab, plus looking over tutorials I'm starting to learn a little bit so if you think that's for the best. Then sir, I will oblige.

Reply:
np, we all started programming from scratch

Reply:
as for the change itself, if you don't strongly feel that i looks better as if (is_test_map) ... then just leave it as it is. As I've written, it's a very minor stylistic change